connectorx/src/sources/postgres/mod.rs

```diff
@@ -199,7 +205,13 @@ where
     fn partition(self) -> Vec<Self::Partition> {
         let mut ret = vec![];
         for query in self.queries {
-            let conn = self.pool.get()?;
+            let mut conn = self.pool.get()?;
+
+            if let Some(queries) = &self.pre_execution_queries {
+                for query in queries {
```

Comment:
rename the inner `query` (e.g., to `pre_query`) so that it won't be confused with the outer `query` variable?